### DISCUSSION TITLE
Question 85: photos with usage restrictions, part II [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#675] [Q&A] [answered]
Description: This is a follow up to question 54. We have some photos from a research project that have specific usage restrictions. I created Event irn 8991 and attached the 3 documents we have on event tab 2, associated records. Does this event, and the 3 MM records, look correct? I tried to follow the docs linked in the response to question 54, but these documents aren’t exactly media agreements so I want to make sure they look ok. 

Next, how do I properly set up the images? Rights irn = 36, Record status = Restricted? Or do I need to create a new rights record? 

The IRB states: “We have a strict protocol for access to these records, requiring the approval of the curator (in this case the P.I. of this project), and with safeguards to protect subject privacy. The curators regularly review and oversee all photographic/visual material held in the Field Museum archives and collections and do not permit their use if deemed sensitive or violating of the privacy of the subjects.” 

How should I document approval by the curator?

**Comment by rondlg** [comment]
Response:
- Rights irn 36 + Record Status ‘Restricted’ is appropriate
> Did the project define a different copyright? (e.g. not © FMNH?)

- To document approval – may need to modify some of these steps to include ‘approval/approver’, but Consultations includes relevant fields: https://docs.google.com/document/d/1WTQlsqCuiir4Xckf_vuQSgS6HEqRvI-E4eDtrhrFDtY/edit#heading=h.3odbkwd4qqfi 
- - Record Status = one of the following
- - - Requires Approval (+ something scary thumbnail) = we know who we need approval from
- - - Restricted = we don’t know who, but need to generally restrict re/use of media

- How to document non-standard agreement / if needed: https://docs.google.com/document/d/1WU88AGbzvvcb6uheUfmFaUbJ2MD6FzNdbYrV7cy8DL4/edit#heading=h.2emme7f7x25y 

> Pandemic Collection IRB approval docs [ED]
> Narratives / Interview
> Attach project’s Event record on the Narrative’s Event tab
> Attach IRB media records to project Event record

**Reply by zkachian** [reply]
A follow up question to this. We also have a PDF of field notes that correspond to these photos. Field notes are different for each group of photos, so where should I attach the MM record of the field notes? To each individual photo it describes, to the group of photos, or somewhere else?

**Comment by rondlg** [comment, marked as answer]
Start by attaching to the event. Future projects may choose to transcribe in a more detailed fashion later.
